ui/app/AppLayouts/Communities/controls/BannerPicker.qml

```diff
@@ -32,15 +32,27 @@ Item {
 
         anchors.fill: parent
 
-        spacing: 8
+        spacing: 19
 
-        StatusBaseText {
-            text: qsTr("Community banner")
+        RowLayout {
+            StatusBaseText {
+                text: qsTr("Community banner")
 
-            font.pixelSize: 15
-            color: Theme.palette.directColor1
+                font.pixelSize: Theme.primaryTextFontSize
+                color: Theme.palette.directColor1
+            }
+            Item {
+                Layout.fillWidth: true
+            }
+            StatusBaseText {
+                text: qsTr("max size 450kb")
```

Comment:
Do we enforce that somehow somewhere? :thinking:

Reply:
Good point! Worth checking, to create a task if we don't.

Reply:
It's enforced in status-go. But the strange thing is it's not max 450kb at image selection time. It's just resized to 450kb in status-go

Reply:
That's even better 😄

Reply:
OK, so this string can be removed then? (or at least reworded since from the user's POV, any image can be uploaded)

Reply:
And it's gone!
Thanks for spotting this out! 🍻